File: Level2/주식가격.py
prices = [1, 2, 3, 2, 3, 1]


def solution(prices):
    answer = []
    for i in range(len(prices)):
        count = 0
        if i != len(prices) - 1:
            for k in range(i+1, len(prices)):
                if prices[k] >= prices[i]:
                    count += 1
                if prices[k] < prices[i]: #가격이 prices[i]보다 낮은것이 나오면 카운트를 세어주고 즉시 break
                    count += 1
                    break
            answer.append(count)
        else:
            answer.append(0)
    return answer

did = solution(prices)
print(did)

File: Level2/주식가격.cpp
#include <string>
#include <vector>
#include <iostream>
using namespace std;

vector<int> solution(vector<int> prices) {
	vector<int> answer;
	int count = 0;
	for (int i = 0; i < prices.size() - 1; i++)
	{
		for (int k = i + 1; k < prices.size(); k++)
		{
			count++;
			if (prices[i] > prices[k])
				break;
		}
		answer.push_back(count);
		count = 0;
	}
	answer.push_back(0);
	return answer;
}